draft-ietf-webpush-protocol.xml

```diff
@@ -217,6 +216,13 @@
           </list>
         </t>
         <t>
+          Examples do not demonstrate the use of HTTP alternative services (<xref target="connecting"/>),
+          which would minimally impact the example exchanges between the User Agent and the Push Service.
+          The one minor, visible change would be the addition of the Alt-Used header (<xref target="RFC7838"/>)
+          in requests from the User Agent to the Push Service. For simplicity, this header has been
+          omitted from examples.
+        </t>

```

Comment:
Yes - I was attempting to avoid redundant ideas with Section 3. Let me make another pass.